setup.py

```diff
@@ -27,4 +27,5 @@
                'sdist': {'dist_dir': '_dist'}},
 
       install_requires=['isodate >= 0.4.9',
-                        'requests >= 2.0.0'],)
+                        'requests >= 2.0.0',
```

Comment:
I read on pypi that minimum tested version of requests was 2.1.0 so would not it be better to raise the version for safety?